doc/locale/ja/LC_MESSAGES/reference.po

```diff
@@ -17566,6 +17566,115 @@ msgid "See :doc:`groonga` to know about supported log flags."
 msgstr ""
 "サポートされているログフラグについては :doc:`groonga` を参照してください。"
 
+msgid "``--since``"
+msgstr ""
+
+msgid ""
+"It specifies the object's modified time which should be checked. If object's "
+"modified time is newer than the specified time, these objects are checked by "
+"``grndb``. You can specify the modified time as ISO 8601 format or ``-"
+"NUNIT`` format such as -3days or -2.5weeks format."
+msgstr ""
+"チェックすべきオブジェクトの更新時刻を指定します。もしオブジェクトの更新時刻"
+"が指定したものよりも新しければ、 ``grndb`` のチェックの対象となります。更新時"
+"刻は ISO 8601形式か -3days や -2.5weeksといった ``-NUNIT`` 形式で指定します。"
+
+msgid ""
+"Here is an example that specifies ``--since`` option in ISO 8601 format::"
+msgstr "以下は ``--since`` オプションをISO 8601形式で指定する例です。"
+
+msgid ""
+"In above example, the objects which are modified after "
+"``2019-06-24T18:16:22`` are checked."
+msgstr ""
+"上記の例では ``2019-06-24T18:16:22`` 以降に更新されたオブジェクトをチェックし"
+"ます。"
+
+msgid ""
+"Here is an example that specifies ``--since`` option in ``-NUNIT`` format::"
+msgstr "以下は ``--since`` オプションを ``-NUNIT`` 形式で指定する例です。"
+
+msgid ""
+"In above example, the objects which are modified in recent 7 days are "
+"checked."
+msgstr "上記の例では、直近7日で更新されたものがチェックされます。"
+
+msgid "``-NUNIT`` accepts the following suffix as a unit."
+msgstr "``-NUNIT`` は以下の単位を受け付けます。"
+
+msgid "Supported suffix"
+msgstr "対応している単位"
+
+msgid "``s``, ``sec``, ``secs``, ``second``, ``seconds``"
+msgstr ""
+
+msgid ""
+"Specify recent N seconds. For example, ``--since=-100s`` means within recent "
```

Comment:
Is `--since=-100s` include 100[sec]  or exclude 100[sec]?

Reply:
It includes 100sec so I wrote `within`.